test/testEth.ts

```diff
@@ -272,58 +282,65 @@ describe('Test new transaction types',  () => {
       value: 100,
       data: '0xdeadbeef',
       accessList: [
-        { 
-          address: '0xe242e54155b1abc71fc118065270cecaaf8b7768', 
+        {
+          address: '0xe242e54155b1abc71fc118065270cecaaf8b7768',
           storageKeys: [
-            '0x7154f8b310ad6ce97ce3b15e3419d9863865dfe2d8635802f7f4a52a206255a6'
-          ]
+            '0x7154f8b310ad6ce97ce3b15e3419d9863865dfe2d8635802f7f4a52a206255a6',
+          ],
         },
-        { 
-          address: '0xe0f8ff08ef0242c461da688b8b85e438db724860', 
-          storageKeys: []
-        }
-      ]
+        {
+          address: '0xe0f8ff08ef0242c461da688b8b85e438db724860',
+          storageKeys: [],
+        },
+      ],
     };
-    await testTxPass(buildTxReq(txData))
-  })
-
-})
+    await testTxPass(buildTxReq(txData));
+  });
+});
 
 if (!process.env.skip) {
   describe('Test ETH Tx Params', () => {
     beforeEach(() => {
-      expect(foundError).to.equal(false, 'Error found in prior test. Aborting.');
-      setTimeout(() => {}, 5000);
-    })
+      expect(foundError).to.equal(
+        false,
+        'Error found in prior test. Aborting.'
+      );
+      setTimeout(() => { return undefined }, 5000);
+    });
 
     it('Should test and validate signatures from shorter derivation paths', async () => {
-      if (constants.getFwVersionConst(client.fwVersion).varAddrPathSzAllowed) {
+      if (getFwVersionConst(client.fwVersion).varAddrPathSzAllowed) {
         // m/44'/60'/0'/x
-        const path = [helpers.BTC_PURPOSE_P2PKH, helpers.ETH_COIN, HARDENED_OFFSET, 0];
+        const path = [
+          helpers.BTC_PURPOSE_P2PKH,
+          helpers.ETH_COIN,
+          HARDENED_OFFSET,
+          0,
+        ];
         const txData = JSON.parse(JSON.stringify(defaultTxData));
         await testTxPass(buildTxReq(txData, 1, path));
-        await testTxPass(buildTxReq(txData, 1, path.slice(0, 3)));      
+        await testTxPass(buildTxReq(txData, 1, path.slice(0, 3)));
         await testTxPass(buildTxReq(txData, 1, path.slice(0, 2)));
-        await testTxFail(buildTxReq(txData, 1, path.slice(0, 1)));            
+        await testTxFail(buildTxReq(txData, 1, path.slice(0, 1)));
       }
-    })
+    });
     it('Should test named units', async () => {
       const txData = JSON.parse(JSON.stringify(defaultTxData));
-      await testTxPass(buildTxReq(txData, `0x${(137).toString(16)}`));
-      await testTxPass(buildTxReq(txData, `0x${(56).toString(16)}`));
-      await testTxPass(buildTxReq(txData, `0x${(43114).toString(16)}`));      
-    })
+      await testTxPass(buildTxReq(txData, `0x${(137).toString(16)}` as unknown as number));
```

Comment:
Are you sure this works?

Reply:
Yeah, `as unknown` wipes out the type information before it's reassigned to number. 

Instead of doing it this way, I updated the function to also accept a string.